src/Source/VectorTilesSource.js

```diff
@@ -112,20 +121,18 @@ class VectorTilesSource extends TMSSource {
                         id: layer.id,
                         order,
                         filterExpression: featureFilter(layer.filter),
-                        zoom: {
-                            min: layer.minzoom || 0,
-                            max: layer.maxzoom || 24,
-                        },
                     });
                 }
             });
 
             if (this.url == '.') {
-                const TMSUrlList = Object.values(style.sources).map((sourceVT) => {
+                const TMSUrlList = Object.values(mvtStyle.sources).map((sourceVT) => {
                     if (sourceVT.url) {
```

Comment:
these default values are not needed anymore ?

Reply:
I don't really understand your question. I only renamed the variable to avoid confusion with itowns.style and itowns.layer.